basics/a87.go

```diff
@@ -0,0 +1,24 @@
+// Simple Input scanner. Reads input from stdin
+// Press Ctrl-D to exit 
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func main() {
+	str := "Enter Name: "
+	name := ""
+	fmt.Println(str)
+	input := bufio.NewScanner(os.Stdin)
+	for input.Scan() {
+		if input.Text() == "" {
+			break
+		} else {
+			name = input.Text()
+		}
+	}
+	fmt.Println(name)
```

Comment:
This is not printed. The terminal just hangs in there.

Reply:
You will have to do ctrl-d to exit the terminal

Reply:
You will have to do ctrl-d to exit the terminal